src/psyclone/gocean1p0.py

```diff
@@ -2466,11 +2450,12 @@ def __init__(self, arg, kernel_call):
         api_config = Config.get().api_conf("gocean1.0")
         try:
             deref_name = api_config.grid_properties[arg.grid_prop].fortran
-        except KeyError:
+        except KeyError as err:
             all_keys = str(api_config.grid_properties.keys())
-            raise GenerationError("Unrecognised grid property specified. "
-                                  "Expected one of {0} but found '{1}'".
-                                  format(all_keys, arg.grid_prop))
+            six.raise_from(GenerationError("Unrecognised grid property "
+                                  "specified. Expected one of {0} but found "
```

Comment:
pycodestyle complains about the indenting.

Reply:
Done.